types/wnumb/index.d.ts

```diff
@@ -1,8 +1,11 @@
-// Type definitions for wnumb 1.0
+// Type definitions for wnumb 1.2.0
 // Project: https://github.com/leongersen/wnumb
-// Definitions by: Corey Jepperson <https://github.com/acoreyj>
+// Definitions by: Corey Jepperson <https://github.com/acoreyj> and Jamie Neubert Pedersen <https://github.com/eikooc>
```

Comment:
Can you follow the readme guide? (append via comma separated entry or wrap into new line):
https://github.com/DefinitelyTyped/DefinitelyTyped#edit-an-existing-package

Reply:
Alright :)

Reply:
@peterblazejewicz everything should be in order now